pkg/tarmak/terraform.go

```diff
@@ -179,6 +179,13 @@ func (c *CmdTerraform) Destroy() error {
 		return err
 	}
 
+	c.tarmak.cluster.Log().Info("Removing environment")
+
+	err = c.tarmak.environment.Remove()
```

Comment:
Not sure if we have to make this command part of `tarmak cluster destroy`. Input is welcome for this one!

Reply:
ok